services/crypto_get_info.proto

```diff
@@ -55,119 +67,219 @@ message CryptoGetInfoQuery {
+     * a means to optionally ensure important accounts remain active.
+     *
+     * ### Staking
+     * Accounts may participate in securing the network by "staking" the account
+     * balances to a particular network node, and receive a portion of network
+     * fees as a reward. An account may optionally decline these rewards but
+     * still stake its balances.
+     *
+     * #### Transfer Restrictions
+     * An account may optionally require that inbound transfer transactions be
+     * signed by that account as receiver (in addition to any other signatures
+     * required, including sender).
+     *
+     */
     message AccountInfo {
+        // Removed prior to oldest available history
+        reserved 5;
+
         /**
-         * The account ID for which this information applies
+         * a unique identifier for this account.
+         * <p>
+         * An account identifier, when assigned to this field, SHALL be of
+         * the form `shard.realm.number`.
          */
         AccountID accountID = 1;
 
         /**
-         * The Contract Account ID comprising of both the contract instance and the cryptocurrency
-         * account owned by the contract instance, in the format used by Solidity
+         * A Solidity ID.
+         * <p>
+         * This SHALL be populated if this account is a smart contract, and
+         * SHALL NOT be populated otherwise.<br/>
+         * This SHALL be formatted as a string according to Solidity ID
+         * standards.
          */
         string contractAccountID = 2;
 
         /**
-         * If true, then this account has been deleted, it will disappear when it expires, and all
-         * transactions for it will fail except the transaction to extend its expiration date
+         * A boolean indicating that this account is deleted.
+         * <p>
+         * Any transaction involving a deleted account SHALL fail.
          */
         bool deleted = 3;
 
         /**
-         * [Deprecated] The Account ID of the account to which this is proxy staked. If proxyAccountID is null,
-         * or is an invalid account, or is an account that isn't a node, then this account is
-         * automatically proxy staked to a node chosen by the network, but without earning payments.
-         * If the proxyAccountID account refuses to accept proxy staking , or if it is not currently
-         * running a node, then it will behave as if proxyAccountID was null.
+         * Replaced by StakingInfo.<br/>
+         * ID of the account to which this account is staking its balances. If
+         * this account is not currently staking its balances, then this field,
+         * if set, SHALL be the sentinel value of `0.0.0`.
          */
         AccountID proxyAccountID = 4 [deprecated = true];
 
         /**
-         * The total number of tinybars proxy staked to this account
+         * The total amount of tinybar proxy staked to this account.
          */
         int64 proxyReceived = 6;
```

Comment:
This is also deprecated I think, atleast not used

Reply:
Will Fix.